packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -24,41 +27,59 @@ export const getComponentCss = (
     '@global': {
       ':host': {
         display: 'block',
-        ...(!compact && {
-          '&(:first-of-type) .root': {
-            borderTop: border,
-          },
-        }),
+        ...hostHiddenStyles,
       },
       button: {
         display: 'flex',
+        position: 'relative',
         justifyContent: 'space-between',
-        margin: `${pxToRemWithUnit(4)} 0`,
+        margin: '2px 0',
         width: '100%',
         textDecoration: 'none',
         border: 0,
+        outline: 0,
         background: 'transparent',
         cursor: 'pointer',
-        transition: getTransition('color'),
-        overflow: 'hidden', // fixes rotating icon to increase bounding box of focus outline in firefox
         textAlign: 'left',
         color: primaryColor,
         ...textSmallStyle,
         fontWeight: fontWeight.semiBold,
-        ...(compact
-          ? { padding: `${pxToRemWithUnit(4)} 0` }
-          : buildResponsiveStyles(size, (s: AccordionSize) => ({
-              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-              // @ts-ignore
-              ...fontSizeText[s],
-              padding: `${pxToRemWithUnit(s === 'medium' ? 20 : 12)} 0`,
-            }))),
-        ...getFocusJssStyle({ color: focusColor }),
+        ...buildResponsiveStyles(size, (s: AccordionSize) => ({
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          fontSize: fontSizeText[s],
+          padding: compact ? '4px 0' : `${s === 'medium' ? '20px' : '12px'} 0`,
+        })),
+        '&::before': {
+          content: '""',
+          position: 'absolute',
+          borderRadius: borderRadiusSmall,
+          transition: getTransition('background-color'),
```

Comment:
Only relevant for hover? So why not move it into the hover media query.

Reply:
You want the transition on hover and when the mouse moves away from the element. If we put it into the hover media query we only have the transition on hover, not when it moves out.

Reply:
No, that's the case when you put it into the `:hover` pseudo selector. The hover media query is supposed to wrap the pseudo selector and should also wrap the transition.